packages/lists/init.lua

```diff
@@ -370,21 +376,23 @@ The way they do is best illustrated by an example.
 \end{enumerate}
 
 \smallskip
-
-\em{Vertical spaces.}
+\noindent
+\em{Vertical spaces}
 \novbreak
 
+\indent
 The package tries to ensure a paragraph is enforced before and after a list.
 In most cases, this implies paragraph skips to be inserted, with the usual \autodoc:setting{document.parskip} glue, whatever value it has at these points in the surrounding context of your document.
 Between list items, however, the paragraph skip is switched to the value of the \autodoc:setting{lists.parskip} setting.
 
 \smallskip
-
-\em{Other considerations.}
+\noindent
+\em{Other considerations}
```

Comment:
For the fun of it, an interesting case where French typographers would likely disagree. (Likely = depends on whom you ask, lol).

Reply:
This actually brings up something I noticed: both the `lists` package and the `math` package have this sort of heading. I'd call it a sort of section, yet perhaps not one that should be numbered or in the TOC. As \subsubsection isn't being used in the docs, I wonder whether this could be defined (in `macros.sil`) to do the 'right' thing consistently?

Reply:
IMHO,we can't use `\sub(sub)section` (or any book sectioning level) as we have no idea where the package documentation would end up (without making assumptions on how the manual is organized). I'd call that a "sort of simple heading", and maybe it could be provided by autodoc.

Reply:
I'm leaving this comment unresolved because I do think there is room for improvement here, but I'm not waiting for it to be part of this PR. If anybody wants to pop open a new PR or issue go for it.